### ISSUE TITLE
the profermance on Stanford Cars is less than 70%.

### ISSUE THREAD

**Issue by zhwzhao** [GuYuc/WS-DAN.PyTorch#26]
I train the network on the stanford cars, the loss is keeping 1.2  when it runs about 90 epoch , the  accuracy on the test dataset is only less than 70%.

the parameter I used is : batch size=4, pretrainmode=resnet50, others are not change.
